Specs/Scene/Cesium3DTileFeatureSpec.js

```diff
@@ -85,9 +85,9 @@ describe(
       });
     });
 
-    describe("3DTILES_metadata", function () {
+    describe("metadata", function () {
```

Comment:
Hm, this test structure feels like it's nested a little too much.

![image](https://user-images.githubusercontent.com/8422414/156422931-0c6cfd7b-470c-4186-a5b6-28c351ce24dd.png)


Maybe the organization should be like this:

* `3D Tiles 1.1 metadata` 
* `3DTILES_metadata` (as a sibling, not a child)
* `3DTILES_implicit_tiling metadata` (explicitly add "metadata" to make it clear)
  * subtree metadata
  * implicit content metadata

Reply:
Essentially just close the `metadata` describe() block after the two tests